app/components/PinListItem.js

```diff
@@ -119,4 +127,7 @@ const style = StyleSheet.create({
     color: 'black',
     alignSelf: 'center',
   },
+  target: {
+    backgroundColor: 'red',
```

Comment:
We can change it when we go through and do polish and styling. Solidify the color scheme and whatnot. I wanted to demonstrate how one would differentiate the target pin in both the map and list contexts, but the actual color or design will have to be done by whoever handles that issue.